<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateDatabase</class>
 <widget class="QWidget" name="CreateDatabase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1011</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>py_face_recognition DataRecord</string>
  </property>
  <widget class="QGroupBox" name="faceRecordGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>641</width>
     <height>581</height>
    </rect>
   </property>
   <property name="title">
    <string>Visualization</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="faceDetectCaptureLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>&lt;font color=red&gt;The camera is closed.&lt;/font&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>21</y>
      <width>641</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="controlHorizontalLayout">
     <item>
      <layout class="QHBoxLayout" name="webcamHorizontalLayout">
       <item>
        <widget class="QPushButton" name="startWebcamButton">
         <property name="text">
          <string>Open the camera</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="enableFaceDetectButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Enable face detection</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>layoutWidget_2</zorder>
   <zorder>faceDetectCaptureLabel</zorder>
   <zorder>layoutWidget</zorder>
  </widget>
  <widget class="QGroupBox" name="dataGroupBox">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>10</y>
     <width>341</width>
     <height>581</height>
    </rect>
   </property>
   <property name="title">
    <string>Steps for data entry</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
      <widget class="QGroupBox" name="dbGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>321</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Step 1: Pick a frame</string>
    </property>
    <widget class="QWidget" name="layoutWidget_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="captureFrameButton">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>20</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Capture current frame</string>
     </property>
    </widget>    
    <widget class="QPushButton" name="releaseFrameButton">
    <property name="geometry">
        <rect>
        <x>21</x>
        <y>55</y>
        <width>281</width>
        <height>31</height>
        </rect>        
    </property>
    <property name="text">
        <string>Release the frame</string>
    </property>        
    </widget>    
   </widget>
   <widget class="QGroupBox" name="userInfoGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>321</width>
      <height>120</height>
     </rect>
    </property>
    <property name="title">
     <string>Step 2: Enter the name</string>
    </property> 
     <widget class="QWidget" name="layoutWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>281</width>
        <height>91</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="infoConfirmHorizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="keyVerticalLayout">
         <item>
          <widget class="QLabel" name="stuIDLabel">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="text">
            <string>Box id：</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="cnNameLabel">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="text">
            <string>Person name：</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="valueVerticalLayout">
         <item>
          <widget class="QLineEdit" name="BoxIDLineEdit">
           <property name="focusPolicy">
            <enum>Qt::StrongFocus</enum>
           </property>
           <property name="readOnly">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="PersonNameLineEdit">
           <property name="focusPolicy">
            <enum>Qt::StrongFocus</enum>
           </property>
           <property name="readOnly">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
   </widget>
   <widget class="QGroupBox" name="userInfoGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>321</width>
      <height>70</height>
     </rect>
    </property>
    <property name="title">
     <string>Step 3: Import to database</string>
    </property> 
    <widget class="QPushButton" name="ImportToDatabaseButton">
    <property name="geometry">
        <rect>
        <x>21</x>
        <y>20</y>
        <width>281</width>
        <height>31</height>
        </rect>        
    </property>
    <property name="text">
        <string>Import to database</string>
    </property>        
    </widget>  
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
